components/buttonSelectors/NumberButtonSelector.js

```diff
@@ -26,7 +39,19 @@ const NumberButtonSelector = (props) => {
         } else{
             updateResponse && updateResponse({id, question: currId, selection: val})
         }
-        submitFunction(val);
+        submitFunction({id, question: currId, selection: val});
+    }
+
+    if(!existingData) {
+        status = 'danger'
+    } else {
+        if(!existingData[currId]) {
+            status = 'danger';
+        } else if(existingData[currId] != selection) {
+            status = 'danger'
+        } else {
+            status = 'success'
+        }
```

Comment:
this is less of a comment and more of a question: i'm assuming that this is needed for the fake validation that's in place right now that changes the field from red to green?